driver-examples/src/main/java/com/datastax/driver/examples/opentelemetry/ZipkinConfiguration.java

```diff
@@ -1,13 +1,63 @@
 package com.datastax.driver.examples.opentelemetry;
 
 import com.datastax.driver.core.Cluster;
-import com.datastax.driver.core.OpenTelemetryConfiguration;
 import com.datastax.driver.core.Session;
+import com.datastax.driver.core.tracing.TracingInfoFactory;
+import com.datastax.driver.opentelemetry.OpenTelemetryTracingInfoFactory;
 import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.common.Attributes;
 import io.opentelemetry.api.trace.Span;
 import io.opentelemetry.api.trace.Tracer;
 import io.opentelemetry.context.Scope;
-import io.opentelemetry.extension.noopapi.NoopOpenTelemetry;
+import io.opentelemetry.exporter.zipkin.ZipkinSpanExporter;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.resources.Resource;
+import io.opentelemetry.sdk.trace.SdkTracerProvider;
+import io.opentelemetry.sdk.trace.export.SimpleSpanProcessor;
+import io.opentelemetry.sdk.trace.export.SpanExporter;
+import io.opentelemetry.semconv.resource.attributes.ResourceAttributes;
+
+class OpenTelemetryConfiguration {
```

Comment:
We may consider moving `OpenTelemetryConfiguration` class with method `initialize(SpanExporter)` to a separate file. This method may be used with various span exporters for various backends. I'd keep here only `ZipkinConfiguration` class with initialization specific for Zipkin.

Reply:
I would also move code that creates keyspace and tables, and loads some data into them to examples, cause now what `ZipkinConfiguration` does isn't really configuration, but example of usage.